theme/templates/resource-landing-page/title-section.html

```diff
@@ -328,6 +363,54 @@ <h2 id="resource-title">{{ title }}</h2>
             margin-top: 1em;
         }
     </style>
+
+    <script>
+        let nonPreferredPathDiv = $("#non-preferred-paths");
+        nonPreferredPathDiv.hide();
+        if (CAN_CHANGE && !RESOURCE_PUBLISHED_OR_UNDER_REVIEW) {
+            let url = "/hsapi/_internal/" + SHORT_ID + "/non-preferred-paths/";
+
+            $.ajax({
+                type: "GET",
+                url: url,
+                dataType: 'json',
+                data: {},
+                async: true,
+                success: function (result) {
+                    $("#checking-for-paths").hide();
+                    result = JSON.stringify(result);
+                    let json_response = JSON.parse(result);
+                    if (json_response.status === "SUCCESS") {
+                        //$("#checking-for-paths").hide();
+                        if (json_response.non_preferred_paths.length) {
+                            nonPreferredPathDiv.show();
+                            let modalBody = nonPreferredPathDiv.find(".modal-body");
+                            let list = document.createElement("UL");
+                            for (let i = 0; i < json_response.non_preferred_paths.length; i++) {
+                                let listItem = document.createElement("LI");
+                                let code = document.createElement("CODE");
```

Comment:
maybe add a custom class to either the LI or the CODE elements to make them easier to select with css?

Reply:
You mean the user will be selecting the path?

Reply:
sorry just saw this note now. I think you resolved this with your `.path-align` class--nice!